metricbeat/helper/http.go

```diff
@@ -22,10 +25,42 @@ type HTTP struct {
 
 // NewHTTP creates new http helper
 func NewHTTP(base mb.BaseMetricSet) *HTTP {
+	config := struct {
+		TLS     *outputs.TLSConfig `config:"ssl"`
```

Comment:
@urso as we start to using this in other places, we should probably move it out from the output package to ?